src/components/elements/favorites-list.tsx

```diff
@@ -33,53 +37,68 @@ const ShareButtons = () => {
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Text
-      </button>
-      <button
-        className="text-center font-semibold text-archway-dark no-underline hocus:underline"
-        onClick={() => (location.href = emailCopy)}
-        data-course-shared="Favorites"
-        disabled={!favs.length}
-      >
-        <EnvelopeIcon
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Email
-      </button>
-      <button
-        className="text-center font-semibold hocus:underline"
-        onClick={() => copyToClip(copyUrl)}
-        data-course-shared="Favorites"
-        disabled={!favs.length}
-      >
-        <ClipboardDocumentIcon
-          width={30}
-          className={twMerge(
-            "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
-            clsx({"bg-black-40": !favs.length})
-          )}
-        />
-        Copy
-      </button>
-    </div>
+    <>
+      <div className="mt-12 flex flex-row justify-between gap-5 *:max-w-60 *:flex-1 *:shrink-0 @md:gap-20">
+        <button
+          className="text-center font-semibold text-archway-dark no-underline hocus:underline"
+          onClick={() => (location.href = smsCopy)}
+          data-course-shared="Favorites"
+          disabled={!favs.length}
+        >
+          <ChatBubbleLeftEllipsisIcon
+            width={30}
+            className={twMerge(
+              "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
+              clsx({"bg-black-40": !favs.length})
+            )}
+          />
+          Text
+        </button>
+        <button
+          className="text-center font-semibold text-archway-dark no-underline hocus:underline"
+          onClick={() => (location.href = emailCopy)}
+          data-course-shared="Favorites"
+          disabled={!favs.length}
+        >
+          <EnvelopeIcon
+            width={30}
+            className={twMerge(
+              "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
+              clsx({"bg-black-40": !favs.length})
+            )}
+          />
+          Email
+        </button>
+        <button
+          className="text-center font-semibold hocus:underline"
+          onClick={() => copyToClip(copyUrl)}
+          data-course-shared="Favorites"
+          disabled={!favs.length}
+        >
+          <ClipboardDocumentIcon
+            width={30}
+            className={twMerge(
+              "mb-4 block w-full rounded-full bg-spirited-dark p-5 text-white",
+              clsx({"bg-black-40": !favs.length})
+            )}
+          />
+          Copy
+        </button>
+      </div>
+      <ToastContainer
```

Comment:
Now that I think about this, we probably can put the container into the layout itself... try this
create a simple component in the `src/components/elements` called `ToastMessage`. Make sure to put `"use client"` at the top of the file.
Put this toast container code into that.
then put the `<ToastMessage/>` into the layout.tsx file.

I believe that will allow us to reuse the same toast message across the whole site.

Reply:
@pookmish Ok.  That makes sense to me. I'm curious why the camel case the `ToastMessage.tsx` file when all the others are lower case?

Reply:
No i'm saying the component would be `ToastMessage`. the file would follow the pattern like `toast-message.tsx`

Reply:
More coffee needed. 👍🏼